tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/Exceptions/CommandResultException.cs

```diff
@@ -1,46 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
-using System.Runtime.Serialization;
-using System.Threading.Tasks;
-
-namespace Azure.Sdk.Tools.TestProxy.Common.Exceptions
```

Comment:
Was this just a random file that I added erroneously and you're cleaning this up here?

Reply:
@scbedd this wasn't a random file that I'm cleaning up. Here are the two exception classes
GitProcessException was in [HttpException.cs](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/Exceptions/HttpException.cs#L11)
HttpException was in [CommandResultException.cs](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/Exceptions/CommandResultException.cs#L10)
I wasn't cleaning, I was actually fixing these and putting them into their correct places.